bedrock/base/templates/macros-protocol.html

```diff
@@ -66,8 +66,9 @@
 
 
 {# Split: https://protocol.mozilla.org/patterns/organisms/split.html #}
-{% macro split(block_class=None, theme_class=None, body_class=None, image_url=None, media_class=None, include_highres_image=False, l10n_image=False, image_alt='') -%}
-<section class="mzp-c-split{% if block_class %} {{ block_class }}{% endif %}">
+{% macro split(id=None, block_class=None, theme_class=None, body_class=None, image_url=None, media_class=None, include_highres_image=False, l10n_image=False, image_alt='') -%}
+
+<section {% if id %} id={{ id }} {% endif %} class="mzp-c-split{% if block_class %} {{ block_class }}{% endif %}">
```

Comment:
This should include quotes around the value:
```suggestion
<section {% if id %}id="{{ id }}"{% endif %} class="mzp-c-split{% if block_class %} {{ block_class }}{% endif %}">
```